Lab2/lab2_drones_manager_test_grading.h

```diff
@@ -121,11 +121,7 @@ class DronesManagerTest {
         int expected_values[] = {0, 5, 4, 1, 2, 6, 3, 7};
         ASSERT_TRUE(manager.get_size() == 8)
         DronesManager::DroneRecord* n = manager.first;
-        //test
```

Comment:
just checking - this is something you added in?

Reply:
yea I put it in as  test code